bib/cmd/bootc-image-builder/main.go

```diff
@@ -110,6 +110,8 @@ func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest
 	// We might want to change this behaviour in the future to match podman.
 	if !c.Local {
 		pullCmd := exec.Command("podman", "pull", "--arch", c.Architecture.String(), c.Imgref)
+		pullCmd.Stdout = os.Stdout
```

Comment:
This will break the manifest subcommand, I learned this the hard way. However, `stderr` should be just fine, right?

Reply:
Excellent point!

Reply:
I updated this now and also added a short comment about an alternative approach (move it into manifestFromCobra and check what command is used)